doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -56,13 +56,13 @@ Next you need to figure out the alias or database id of the "parent" Dataverse c
   export SERVER_URL=https://demo.dataverse.org
   export PARENT=root
 
-  curl -H X-Dataverse-key:$API_TOKEN -X POST $SERVER_URL/api/dataverses/$PARENT --upload-file dataverse-complete.json
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST "$SERVER_URL/api/dataverses/$PARENT --upload-file dataverse-complete.json"
```

Comment:
should the close-quote here follow $PARENT? (same question applies to other similar changes below)

Reply:
Hello @donsizemore ,
Thank you for this return.
Yes you are right, the close-quote here should follow $PARENT,  it's a mistake on my side. 
I will fix that next Wednesday.

Reply:
@sirineREKIK wonderful, thank you! there are more such instances below in the page.

Reply:
Hello @donsizemore 
I pushed a new commit to fix mistakes.
Regards,